docs/pull-requests.md

```diff
@@ -0,0 +1,161 @@
+
+In this situation, do your best to state your position, providing the author with missing context, and accept that the author has the right to make the final decision.
+
+### Praise good work
+
+If the author of the pull request did a great job, be sure to let them know!
+
+### Take criticism offline
+
+If you fundamentally disagree with the approach that the author has taken, it may be best to have a conversation with them outside of the pull request itself. You might find you can resolve differences more easily and more quickly — while preventing a public heated discussion from making you, the author, or anyone else look bad — by using an alternate form of communication, such as a video call.
+
+### "Request changes" sparingly
+
+When you write a review on GitHub you can assign it to one of three states: "Comment", "Request changes", or "Approved".
+
+The "request changes" state places an X next to your review and prevents the author from merging their changes until either you approve the pull request or someone else dismisses your review. As a result, consider using it in circumstances where you truly feel that the changes being proposed cannot be merged, making sure to explain your reasoning in the review.
+
+## On receiving feedback
+
+### Be open to other perspectives
+
+Understand that some people's approaches may be motivated by a different set of values than you. Their perspective may reveal something you're not seeing.
+
+### Assume positive intent
+
+If you receive a comment that is brusque, has a negative tone, is or difficult to decipher, do your best to handle it gracefully and ask the reviewer for more information if necessary.
+
+### Point to updates
+
+As you are fulfilling others' requests, respond by linking to the commit that contains the changes you made. This gives reviewers the ability to check your work and allows the discussion to reach a resolution. (Tip: You can find commits you've pushed in the "Conversation" view of the pull request. If you copy a commit ID, then GitHub will link to it when you paste it in a comment. You can also just type the commit ID in your comment and GitHub will automatically link it.)
+
+```
+Good catch! Updated in abc1234.
+```
+
+### Employ alternate forms of communication
+
+If you sense that a conversation between you and the author is proceeding in an undesirable direction, consider reaching out to the person directly and offering to talk through your concerns on a video call.
+
+## On maintaining pull requests
+
+### Communicate takeovers
+
+If you need to take over and merge someone else's branch, let them know so they aren't surprised.
+
+### Rebase sparingly
+
+Once you've created a pull request, avoid amending, rebasing, or otherwise altering the history of the branch, but push up each new change as a new commit instead. This has two advantages:
```

Comment:
Nit: Maybe we could say "Once you've marked a pull request as ready for review"? Personally I like the freedom to change a draft PR however I want without affecting reviewers, as I assume that if any review _has_ happened that it has been cursory/high-level and little context will be lost of any.